pages/Signin.vue

```diff
@@ -1,88 +1,105 @@
 <template>
-  <div class="main min-[1120px]:flex min-[1120px]:justify-evenly py-[2rem] px-[0.1rem] mt-[3rem]">
-    <div class="left-panel mt-[2.69rem] max-[1119px]:hidden min-[1120px]:block">
-      <img src="../assets/37.order-delivered-3.png" class = "w-[32rem] h-[28rem] border-1 border-gray-400 min-[1120px]:order-1" alt="delivery-img">
-    </div>
-    <div class="right-panel max-[1119px]:px-[2rem] min-[1120px]:px-[0rem]">
-      <div class="form-container">
-        <form @submit="submitForm">
-          <div class="credentials - container min-[1120px]:order-2 min-[1120px]:mt-[3rem]">
-        <div class="catch-line-outer">
-            <div class="catch-line-inner">
-                <p class = "text-slate-500 font-semibold">"Welcome back! Please enter your details"</p>
-            </div>
+  <div class="max-lg:-mt-6">
+    
+    <div class="main min-[1120px]:flex min-[1120px]:justify-evenly py-[2rem] px-[0.1rem] mt-[1.5rem]">
```

Comment:
@rrtk857 Why are we changing the contents inside the pages, any particular reasone?

Reply:
I have made changes in the files so that pages can fit properly with the navbar. Like I have removed some extra space between pages and navbar. Also in mobile view, content is overlapping with the navbar, so I tried to resolve that by changing some code in files and navbar. Also linked checkout page to the cart pages. Else I have applied formatting. If the changes are not correct please let me know, I will remove it.
![Screenshot (83)](https://github.com/bsoc-bitbyte/GetIt/assets/129523259/ebe6dbd9-de65-4c76-a493-9a101f7c0108)
![Screenshot (82)](https://github.com/bsoc-bitbyte/GetIt/assets/129523259/79795ced-0d04-4168-ae3a-9b899653d190)
![Screenshot (84)](https://github.com/bsoc-bitbyte/GetIt/assets/129523259/06000144-14fb-40f6-b921-b58da60e8094)
![product1](https://github.com/bsoc-bitbyte/GetIt/assets/129523259/ce87a18c-fc78-451f-972d-b587cbdf92ad)
![signin1](https://github.com/bsoc-bitbyte/GetIt/assets/129523259/bef880bb-4893-417d-9052-5ef210607382)
![product](https://github.com/bsoc-bitbyte/GetIt/assets/129523259/79e4e4e8-02eb-44b7-857e-82b1a467d070)
move it.